internal/mtail/basic_tail_integration_test.go

```diff
@@ -4,52 +4,14 @@
 package mtail_test
 
 import (
-	"fmt"
 	"os"
 	"path/filepath"
-	"sync"
 	"testing"
 
 	"github.com/google/mtail/internal/mtail"
 	"github.com/google/mtail/internal/testutil"
 )
 
-func TestBasicTail(t *testing.T) {
```

Comment:
If this test can be made to work on Windows, that would be ideal.  It really just checks that the tailer can open a file, which I expect should also work on Windows.

Reply:
I think this ended up in the other file on accident. Will be reverted on the next push.